src/event_bus.cpp

```diff
@@ -63,6 +63,9 @@ void event_bus::unsubscribe( event_subscriber *s )
 
 void event_bus::send( const cata::event &e ) const
 {
+    if( e.type() == event_type::num_event_types ) {
+        return;
+    }
```

Comment:
Is this here to handle the failure branch from `make_dyn`?  If so, that probably warrants a comment.

And maybe it's worth having a `debugmsg` here too just in case it ends up being called in another situation in the future.